src/commands/index.ts

```diff
@@ -154,9 +154,9 @@ export function load(client: any): any {
   if (experiments.isEnabled("internalframeworks")) {
     client.frameworks = {};
     client.frameworks.stacks = {};
-    client.frameworks.stacks.list = loadCommand("frameworks-stacks-list");
+    client.frameworks.stacks.list = loadCommand("frameworks-backends-list");
```

Comment:
I assume we're not responsible for changing the client.frameworks.stacks.list? Or at least not in this PR?

Reply:
I deleted the file "frameworks-stacks-list.ts" and created "frameworks-backends-list.ts" file. So, we should replace the values here.